testsuite/tests/c-api/alloc_async.ml

```diff
@@ -1,7 +1,5 @@
 (* TEST
    modules = "alloc_async_stubs.c"
-   * skip
-   reason = "alloc async changes: https://github.com/ocaml/ocaml/pull/8897"
```

Comment:
The `reason` line seems to explain why the test is there, rather than why it had been disabled. Let's keep it.

Reply:
`reason` sets the message for the outcome of the test:
https://github.com/ocaml/ocaml/blob/17d9f7cb100f2aff3d0b7d2372d41e239fedcd9c/ocamltest/builtin_variables.ml#L65-L66

It could be moved to a comment?

Reply:
Description added (see below)